azbfs/url_file.go

```diff
@@ -119,8 +128,19 @@ func (f FileURL) GetProperties(ctx context.Context) (*PathGetPropertiesResponse,
 		nil, nil, nil, nil, nil)
 }
 
+func (f FileURL) SetProperties(ctx context.Context, headers BlobFSHTTPHeaders) (*PathUpdateResponse, error) {
```

Comment:
It looks like SetProperties was added because maybe it was going to be used, but in the end its not used. (No usages found at present).  Can we remove it?  (i.e. reverse the addition of it, since its effectively untested code, due to it not being used).

Reply:
I actually added it to just have parity with the API. Certainly can remove it.